src/BenchmarkDotNet/Columns/FirstCallColumn.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using BenchmarkDotNet.Engines;
+using BenchmarkDotNet.Extensions;
+using BenchmarkDotNet.Helpers;
+using BenchmarkDotNet.Horology;
+using BenchmarkDotNet.Reports;
+using BenchmarkDotNet.Running;
+
+namespace BenchmarkDotNet.Columns
+{
+    public class FirstCallColumn : IStatisticColumn
+    {
+        public static readonly IColumn Default = new FirstCallColumn();
+        
+        public string Id => nameof(FirstCallColumn);
+
+        public string ColumnName => "FirstCall";
+
+        public bool AlwaysShow => true;
+
+        public ColumnCategory Category => ColumnCategory.Statistics;
+
+        public int PriorityInCategory => int.MinValue;
+
+        public bool IsNumeric => true;
+
+        public UnitType UnitType => UnitType.Time;
+
+        public string Legend => "Execution time of the first call (Jitting included)";
```

Comment:
I suggest removing the "(Jitting included)" note. It doesn't provide any value for people who are not aware of jitting and may confuse people who know about it (how is it possible to measure the duration of the first call without jitting?). Also, it will be better to write "first method call" or "first benchmark call" instead of just "first call".

Reply:
Good point, without profiler, it's impossible to measure the duration of the first call without jitting. I've removed the text.